src/diffusers/loaders/lora_pipeline.py

```diff
@@ -2276,6 +2276,482 @@ def save_lora_weights(
         )
 
 
+class CogVideoXLoraLoaderMixin(LoraBaseMixin):
+    r"""
+    Load LoRA layers into [`CogVideoXTransformer3DModel`],
+    [`T5EncoderModel`](https://huggingface.co/docs/transformers/model_doc/t5#transformers.T5EncoderModel). Specific to
+    [`CogVideoX`].
+    """
+
+    _lora_loadable_modules = ["transformer", "text_encoder"]
```

Comment:
```suggestion
    _lora_loadable_modules = ["transformer"]
```

For now, since we removed text encoder related training, need to remove everything related to this in the lora loader

Reply:
ok we should remove text_encoder from lora in this PR?

Reply:
So, I tried removing it but this causes test failures in ~10 different places, so I chose not to do it for now since it would require significant modification of many tests. I think it's okay to leave it here in case someone manages to fine tune the text encoder and wants to use it (eventually we can add support too).

Reply:
don't think we should add this code unless it is needed though
I just went through all the loraloadermixin here, I think we currently do not support t5 at all - cc @sayakpaul here to confirm if it is the case?

Reply:
Yes I came to the same conclusion. I'm actually working on removing thet text encoder parts at the moment so will update in a bit.

Need to fight a few more tests :fist_oncoming: